src/app/components/PageLayout/LinkableDiv.tsx

```diff
@@ -25,5 +26,11 @@ export const LinkableDiv: FC<JSX.IntrinsicElements['div'] & { id: string }> = pr
     }
   }, [id, hash, element])
 
-  return <div {...props} ref={divRef} />
+  return (
+    <>
+      {/* Don't scroll the target underneath sticky Header */}
+      <GlobalStyles styles="html { scroll-padding-top: 180px; }" />
```

Comment:
We use index.css as global styles. Why we can't define this CSS prop there? Can this affect layout somehow or it's just preference for this one particular case?